src/components/Navbar.js

```diff
@@ -1,31 +1,40 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
 
 const Navbar = () => {
   const links = [
     {
       id: 1,
       path: '/books',
```

Comment:
- [ ] Nice working on your navigation, Kindly check when the root `/` page loads it shows only the header, you can make the `/books` route the root page `/` to solve this.